src/AutoRest.CSharp/Common/Output/Models/ClientMethod.cs

```diff
@@ -21,5 +21,21 @@ public ClientMethod(string name, RestClientMethod restClientMethod, string? desc
         public string? Description { get; }
         public Diagnostic Diagnostics { get; }
         public string Accessibility { get; }
+
+        public bool IsPut
+        {
+            get
+            {
+                return this.RestClientMethod.IsPut;
+            }
+        }
```

Comment:
We could simplify this 
```suggestion
        public bool IsPut => this.RestClientMethod.IsPut;
```